src/getting_started.md

```diff
@@ -1,6 +1,9 @@
 # Getting Started
 
-This guide walks you through the steps needed to get the Alan compiler and runtimes on your local machine. To work on Alan itself, see [Contributing to Alan](./contributing.md). To learn more about Alan, see [Why we created Alan](https://alan-lang.org/why_alan.html).
+This guide walks you through the steps needed to get the Alan compiler and VM on your local machine. You can also:
+- Read through some idiomatic [Alan examples](./advanced_examples.md)
+- Read more about [Why we created Alan](https://alan-lang.org/why_alan.html)
+- [Contribute to Alan](./contributing.md)
```

Comment:
Maybe a fourth bullet point on using the browser-based compiler and runtime on the homepage?

Reply:
well, the playground is also available in the docs